roofit/roofitcore/src/RooHist.cxx

```diff
@@ -621,7 +592,7 @@ bool RooHist::hasIdenticalBinning(const RooHist& other) const
     GetPoint(i,x1,y1) ;
     other.GetPoint(i,x2,y2) ;
 
-    if (fabs(x1-x2)>1e-10) {
+    if (std::abs(x1-x2)>1e-10) {
```

Comment:
Is the condition correct in case of x1 and x2 are very large ?

Reply:
Thank for spotting this, good point!

Indeed, as the goal of this function is to figure out if two `RooHists` have the same bin centers within some small epsilon, this epsilon should better depend on the overall scale of the x values. I think this is well represented by the average bin width, so I'll change this to:

```c++
if (std::abs(x1-x2) > 1e-10 * _nominalBinWidth) {
```